CHANGES.rst

```diff
@@ -65,6 +65,8 @@ datamodels
 - Added two new header keywords to track the rate of cosmic rays and snowball/showers
   [#7609, spacetelescope/stdatamodels [spacetelescope/stdatamodels#173]
 
+- Remove ``jwst.datamodels.schema`` in favor of ``stdatamodels.schema`` [#7660]
```

Comment:
This entry will need to be moved to the top of the log, now that it's not part of the recent releases that've happened.

Reply:
Thanks! I updated the changelog moving this entry to the top, 'unreleased' version.